src/executables/master_heartbeat.c

```diff
@@ -4893,7 +4920,6 @@ hb_cluster_initialize (const char *nodes, const char *replicas)
 			   hb_Cluster->num_nodes);
```

Comment:
It seems that this debugging log information is not useful. How about removing it together? Below is the remaining master error log from my tests.

```
Time: 05/30/24 11:37:38.472 - ERROR *** file ../src/executables/master_heartbeat.c, line 2779 ERROR CODE = -839, Tran = -1, EID = 2
"ha_node_list is empty.": Unknown system parameter or bad value.

Time: 05/30/24 11:37:38.472 - DEBUG *** file ../src/executables/master_heartbeat.c, line 4920
hb_Cluster->num_nodes is smaller than '1'. (num_nodes=-1).

Time: 05/30/24 11:37:38.472 - DEBUG *** file ../src/executables/master_heartbeat.c, line 5185
hb_cluster_initialize failed. (error=-839).
```
```
Time: 05/30/24 11:42:45.109 - ERROR *** file ../src/executables/master_heartbeat.c, line 2779 ERROR CODE = -839, Tran = -1, EID = 2
"cannot find (hornetmj1) in the ha_node_list.": Unknown system parameter or bad value.

Time: 05/30/24 11:42:45.109 - DEBUG *** file ../src/executables/master_heartbeat.c, line 4920
hb_Cluster->num_nodes is smaller than '1'. (num_nodes=-1).

Time: 05/30/24 11:42:45.109 - DEBUG *** file ../src/executables/master_heartbeat.c, line 5185
hb_cluster_initialize failed. (error=-839).
```

Reply:
`I got it, I will remove debug log that is "hb_Cluster->num_nodes is smaller than '1'. (num_nodes=-1)"`